### PR TITLE
Add postgresql-debversion

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
This is now green, will see about converting to scl

**Comment by mdellweg** [comment]
Thank you very much. We are very happy to see this progressing.

File: packages/katello/postgresql-debversion/postgresql-debversion.spec [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+Group:    Applications/System
+License:  GPLv3
+URL:      https://salsa.debian.org/postgresql/postgresql-debversion
+Source0:  https://salsa.debian.org/postgresql/postgresql-debversion/-/archive/v%{version}/postgresql-debversion-v%{version}.tar.gz
+Patch0:   0001-Copy-relevant-code-from-apt-pkg-to-ease-packaging.patch
+
+# BuildRequires: g++
+BuildRequires: postgresql-devel
+# BuildRequires: postgresql-server-devel
```
Comment (ehelms) [inline packages/katello/postgresql-debversion/postgresql-debversion.spec]: Looks like we can cleanup these commented out build requires.

This is built using postgresql-devel but not the SCL postgresql-devel. Is there anything we need to worry about with differences in versions of Postgresql?

File: packages/katello/postgresql-debversion/postgresql-debversion.spec [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+%files
+%{_libdir}/pgsql/debversion.so
+%{_datadir}/pgsql/extension/debversion--1.0.5--1.0.6.sql
+%{_datadir}/pgsql/extension/debversion--1.0.6--1.0.7.sql
+%{_datadir}/pgsql/extension/debversion--1.0.7--1.0.8.sql
+%{_datadir}/pgsql/extension/debversion--1.0.8--1.1.sql
+%{_datadir}/pgsql/extension/debversion--1.1.sql
+%{_datadir}/pgsql/extension/debversion--unpackaged--1.0.5.sql
+%{_datadir}/pgsql/extension/debversion.control
```
Comment (ehelms) [inline packages/katello/postgresql-debversion/postgresql-debversion.spec]: All of these will land in the system directory right? How will rh-postgresql12 which lives in a different data and libdir know to find them?

File: packages/katello/postgresql-debversion/postgresql-debversion.spec (hunk 1 above)
Comment (jlsherrill) [reply packages/katello/postgresql-debversion/postgresql-debversion.spec]: you might have missed my comment: https://github.com/theforeman/foreman-packaging/pull/4741#issuecomment-589087128

scl-ificiation is still to come

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test rpm]

**Comment by jlsherrill** [comment]
[test katello]

**Comment by jlsherrill** [comment]
@ehelms suggested we not worry about el8 builds at the moment while we wait for the build team to continue their work.

@ehelms @ekohl @evgeni   any other comments before merging?

**Comment by jlsherrill** [comment]
[test rpm]

**Pull request merged**